### ISSUE TITLE
syntax error on install-globaleaks.sh

### ISSUE THREAD

**Issue by larrykind** [globaleaks/GlobaLeaks#3199]
**Describe the bug**
trying to update to 4.8.1 from 4.8.0. The normal apt install globaleaks seems to stop to 4.8.0.
Trying to get install-globaleaks.sh and launching it, I got this error:
`./install-globaleaks.sh: 13: ./install-globaleaks.sh: Syntax error: "(" unexpected`

**Comment by evilaliv3** [comment]
Thank you @larrykind for reporting this.

Can you clarify on which distribution are your running the script and if this is the same system where the version 4.8.0 was already present or you are migrating it?

Thank you

**Comment by larrykind** [comment]
@evilaliv3 I'm running ubuntu 18.04 bionic. 
Yes the system is the same. 
Before I tried `apt install gloabaleaks` but It resulted that 4.8.0 was the latest version available. So I freshly downloaded install-globaleaks.sh and tried to run installation from it, and got the error.

**Comment by evilaliv3** [comment]
@larrykind : would you please check the file? maybe it has not been downloaded correctly and is incomplete?

**Comment by larrykind** [comment]
@evilaliv3 thank you, the problem was due to my firewall configuration. 
Anyway, after install script finished to run I just got this, updating from 4.8.0 to 4.8.1:

```
Running: "apt-get update -y"... SUCCESS 
Running: "apt-get install globaleaks -y"... SUCCESS 
Ouch! The installation is complete but GlobaLeaks failed to start.
```
Restarting globaeaks and it started, but it seems from homepage changelog section to run 4.8.0.

**Comment by evilaliv3** [comment]
Thank you @larrykind.

I notice that by mistake we have not released the new package 4.8.1 for bionic.

I will fix the repository in the next days.

Giovanni
